metadata/service/aggregator/cluster.yml

```diff
@@ -0,0 +1,13 @@
+applications:
```

Comment:
Good point although I didn't see such practice in other formulas. Do you have some examples?